.github/workflows/release-brew.yaml

```diff
@@ -3,21 +3,118 @@ on:
   release:
     types: [created]
 
+env:
+  FORMULA: cbmc-viewer
+  TAP: aws/tap
+  BOT_USER: aws-viewer-for-cbmc-release-ci
+  RELEASE_TAG: ${GITHUB_REF/refs\/tags\/}
+  VERSION: $(echo ${{ github.ref }} | cut -d "/" -f 3 | cut -d "-" -f 2)
+  FORK_REPO: https://${{ secrets.RELEASE_CI_ACCESS_TOKEN }}@github.com/$BOT_USER/homebrew-$(echo $TAP |cut -d "/" -f 2).git
+
 jobs:
   homebrew-pr:
     name: Homebrew Bump Formula PR
     runs-on: macos-10.15
     steps:
-      - name: Get release tag name
-        run: echo "RELEASE_TAG=${GITHUB_REF/refs\/tags\/}" >> $GITHUB_ENV
       - name: Configure git user name and email
         uses: Homebrew/actions/git-user-config@master
         with:
-          username: aws-viewer-for-cbmc-release-ci
+          username: ${{ env.BOT_USER }}
+
       - name: Create homebrew PR
         run: |
+          brew tap ${{ env.TAP }}
           brew update-reset
-          brew tap aws/tap
-          brew bump-formula-pr --tag "$RELEASE_TAG" --revision "$GITHUB_SHA" aws/tap/cbmc-viewer
+          brew bump-formula-pr --tag "${{ env.RELEASE_TAG }}" --revision "$GITHUB_SHA" ${{ env.TAP }}/${{ env.FORMULA }}
         env:
           HOMEBREW_GITHUB_API_TOKEN: ${{ secrets.RELEASE_CI_ACCESS_TOKEN }}
+
+  build-bottle:
+    needs: homebrew-pr
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, macos-10.15]
```

Comment:
Could we also include ubuntu-18.04 since I think Windows Subsystem for Linux is 18 and not yet 20.

Reply:
Both `ubuntu-latest` and `ubuntu-18.04` generates bottle named `cbmc-viewer-VERSION.x86_64_linux.bottle.tar.gz` as there underlying architecture is same - `x86_64_linux` (on which bottle creation depends)

So same bottle will be used for both platforms.